proposals/lip-0041.md

```diff
@@ -170,6 +167,85 @@ This function verifies the nonce of a given transaction `trs` and returns a bool
 
 This function returns `true` if the transaction object contains a valid nonce, `false` otherwise. This is done using the `isValidNonce` function defined [below](#transaction-verification).
 
+### Genesis Block Processing
+
+#### Genesis State Initialization
+
+During the genesis state initialization stage, the following steps are executed. If any step fails, the block is discarded and has no further effect.
+
+Let `genesisBlockAssetBytes` be the bytes included in the block assets for the Auth module and let `genesisBlockAssetObject` be the deserialization of `genesisBlockAssetBytes` according to the `genesisAuthStoreSchema` schema, given below. 
```

Comment:
As it was pointed out in the [LIP 47 review](https://github.com/LiskHQ/lips/pull/119#discussion_r744547641), this is not a strictly correct formulation, since `genesisBlockAssetBytes` should refer to the `data` property of a deserialized entry of the `assets` array.

Reply:
saw that comment... in the Token module, I wrote (probably anything along those lines should work, but the PR is yet to be reviewed):

Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets for the Token module and let `genesisBlockAssetObject` be the deserialization of `genesisBlockAssetBytes` according to the `genesisTokenStoreSchema` schema, given above.

Reply:
edited following Maxime's suggy